src/components/utils/layer-manager/index.ts

```diff
@@ -0,0 +1,9 @@
+export {layerManager, getLayersCount} from './LayerManager';
+export type {
+    LayerConfig,
+    LayerExtendableProps,
+    ContentElement,
```

Comment:
Not sure if we want LayerExtendableProps and ContentElement to be exposed

Reply:
done